Swift/Adhan/Adhan.swift

```diff
@@ -229,28 +224,27 @@ public struct PrayerTimes {
         }
         
         // special case for moonsighting committee above latitude 55
-        if calculationParameters.method == .moonsightingCommittee && coordinates.latitude >= 55 {
+        if calculationParameters.method == .moonsightingCommittee, coordinates.latitude >= 55 {
```

Comment:
I would like to continue to use ampersands for if statements, I feel that commas are primarily for "if let" statements.